### PR TITLE
update Rawls workspace endpoint

### PR DESCRIPTION
**Pull request by aweng98**
This is used by Rawls api tests in [Rawls PR #1070](https://github.com/broadinstitute/rawls/pull/1070)

### CODE REVIEW

File: serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/Rawls.scala [hunk 1]
```diff
@@ -194,6 +194,18 @@ trait Rawls extends RestClient with LazyLogging {
...
+    def getAuthDomainsInWorkspace(namespace: String, name: String)(implicit token: AuthToken): List[String] = {
+      import scala.collection.JavaConverters._
+      val response = getWorkspaceDetails(namespace, name)
+      mapper.readTree(response).at("/workspace/authorizationDomain").findValuesAsText("membersGroupName").asScala.toList
+    }
+
+    def getWorkspaceNames()(implicit token: AuthToken): List[String] = {
+      import scala.collection.JavaConverters._
+      val response = list()
+      mapper.readTree(response).findValuesAsText("name").asScala.toList
```
Comment (marctalbott) [inline serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/Rawls.scala:207]: Does this need to start at `/workspace`? Like this: `mapper.readTree(response).at("/workspace").findValuesAsText("name").asScala.toList`. Check the response payload for `list` https://rawls.dsde-dev.broadinstitute.org/#!/workspaces/listWorkspaces

Comment (aweng98) [reply serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/Rawls.scala:207]: recursive descent lookups starting from the root JsonNode

File: serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/Rawls.scala [hunk 2]
```diff
@@ -194,6 +194,18 @@ trait Rawls extends RestClient with LazyLogging {
       logger.info(s"Updating acl for workspace $name in $namespace")
       patchRequest(url + s"api/workspaces/$namespace/$name/acl?inviteUsersNotFound=$inviteUsersNotFound", aclUpdates.map(e => e.toMap))
     }
+
+    def getAuthDomainsInWorkspace(namespace: String, name: String)(implicit token: AuthToken): List[String] = {
+      import scala.collection.JavaConverters._
+      val response = getWorkspaceDetails(namespace, name)
+      mapper.readTree(response).at("/workspace/authorizationDomain").findValuesAsText("membersGroupName").asScala.toList
```
Comment (aweng98) [inline serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/Rawls.scala:201]: without `at("/workspace/authorizationDomain")` would still work.

File: serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/Rawls.scala (hunk 1 above)
Comment (marctalbott) [reply serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/Rawls.scala:207]: Cool, didn't realize it was recursive

**Review by marctalbott — APPROVED** [review]

**Review by ansingh7115 — APPROVED** [review]

**Pull request merged**